src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/config/WebClientConfiguration.kt

```diff
@@ -85,6 +86,9 @@ class WebClientConfiguration(
     return WebClient.builder().baseUrl(basmRootUri).build()
   }
 
+  @Bean
+  fun prisonRegisterWebClient(): WebClient = WebClient.builder().baseUrl(prisonRegisterApiUrl).build()
```

Comment:
The prison-register end-point /prisons/id/{prisonId} isn't protected so there's no need to configure authentication headers.  This client works for the prison-register api end-points and the health end-point.